tests/src/whisk/core/entity/test/DatastoreTests.scala

```diff
@@ -141,7 +141,7 @@ class DatastoreTests extends FlatSpec
         }
     }
 
-    ignore should "CRD activation with utf8 characters" in {
+    it should "CRD activation with utf8 characters" in {
```

Comment:
This is unrelated to the fix applied in the invoker http call right?
But if it works now let's enable the test I agree.